src/server/player_sao.h

```diff
@@ -235,6 +235,7 @@ struct PlayerHPChangeReason
 	enum Type : u8
 	{
 		SET_HP,
+		SET_HP_MAX,
```

Comment:
should be annotated: `// internal type to allow distinguishing hp reset and damage (for effects)`